highfidelity/hifi
# Allow Hand Animations to be Overridden

This changes the avatar-animation json so that we have override animations for the right and left hand, similar to how we have whole body override animations.

https://highfidelity.manuscript.com/f/cases/21958/Add-MyAvatar-overrideHandAnimation-API

Test Plan:
1) open interface in hmd mode.
2) run the following script http://hifi-content.s3.amazonaws.com/angus/handAnim/overrideHandAnimations.js
3) expected result: you should be able to trigger peace, okay, shaka, horns, and heart on the right and left hands.  The triggers differ in vive and oculus.  Vive controller inputs are very tricky to get to fire, but it should work with touching the thumb in the forward direction and the right direction.  The further trigger is to press the button on the side of the controller facing the user.  The script was developed in vive, but should work in oculus.
